styles.css

```diff
@@ -1063,6 +1149,54 @@ textarea:focus {
   .heading {
     margin-top: 120px;
   }
+
+  .modal-container {
+    background: #fff;
+    border: 1px solid #ebecf0;
+    box-shadow: 48px 48px rgba(37, 47, 137, 0.08);
+    height: 664px;
+  }
+
+  /* .popup-content {
+   display: flex;
+   flex-direction: row;
+   margin-top: 12px;
+   gap: 12px;
+  } */
+
+  .description {
+    width: 424px;
+    height: 336px;
+    font-size: 16px;
+    line-height: 30px;
+    color: #344563;
+    float: right;
+    gap: 12px;
+  }
```

Comment:
- [ ] Like the previous reviewer mentioned please make sure your design looks as close as possible to the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/180230869-fcf27ba8-838d-48a9-a880-ad47fc1a4fc0.png)  |  ![image](https://user-images.githubusercontent.com/74814780/180230955-7aa1ac6d-9522-4d9a-95f5-698f41dc9388.png)